lib/battle_boats/engine.rb

```diff
@@ -4,28 +4,44 @@
 module BattleBoats
   class Engine
     def initialize(interface: BattleBoats::ConsoleUI.new,
-                   board: BattleBoats::Board.new)
+                   enemy_board: BattleBoats::Board.new,
+                   ally_board: BattleBoats::Board.new)
       @interface = interface
-      @board = board
+      @enemy_board = enemy_board
+      @ally_board = ally_board
+    end
+
+    def place_ships_manually
```

Comment:
Could this functionality reasonably exist elsewhere? Could it belong in its own class? Is this conceptually related to `Board#place_ships_randomly`? Should they be more symmetrical? Is this a decision we need to make now?